### PR TITLE
Policies screen top button reworked

### PR DESCRIPTION
**Pull request by Gualdimar**
half solves #8548

### CODE REVIEW

**Review by yairm210 — APPROVED** [review]

**Comment by yairm210** [comment]
Good for me
Waiting for @vegeta1k95 's approval

**Comment by vegeta1k95** [comment]
Looks good. I'll add gradual filling with yello later. This can be merged

**Pull request merged**
